tests/e2e/specs/editor/publishingFlow.js

```diff
@@ -90,4 +94,34 @@ describe('Publishing Flow', () => {
     await expect(page).toMatch('Publishing Flow Test');
     await expect(page).toMatchElement('amp-story-player');
   });
+
+  it('should guide me towards creating a new post to embed my story in classic editor', async () => {
+    await activatePlugin('classic-editor');
+    await createNewStory();
+
+    await expect(page).toMatchElement('input[placeholder="Add title"]');
+    await page.type('input[placeholder="Add title"]', 'Publishing Flow Test');
+
+    // Publish story.
+    await expect(page).toClick('button', { text: 'Publish' });
+
+    await expect(page).toMatchElement('button', { text: 'Dismiss' });
+
+    // Create new post and embed story.
+    await expect(page).toClick('a', { text: 'Add to new post' });
+    await page.waitForNavigation();
+
+    await expect(page).toMatch('Publishing Flow Test');
```

Comment:
This also needs an `expect(await getEditedPostContent()).toMatchSnapshot();` like the above test to actually verify that the shortcode is in the content.

Reply:
Oh, wait, that's a Gutenberg-only method though. Needs some other way to find the post content.

Reply:
I removed this for a reason. There isn't a way easily get text content from the tinymce editor. Isn't the render shotcode the front end enough.

Reply:
It's as simple as `document.getElementById('content').innerHTML` or `document.querySelector('.wp-editor-area').innerHTML` if that's preferable

Reply:
Added a fix here - https://github.com/google/web-stories-wp/pull/4849/commits/89a65e8d2fe10a83131cfc02626b5733561419cc